build-configs/aschaffenburg/index.js

```diff
@@ -0,0 +1,110 @@
+// @flow
+
+import { lightTheme, darkTheme } from './theme'
+import type {
+  AndroidBuildConfigType,
+  CommonBuildConfigType,
+  iOSBuildConfigType,
+  WebBuildConfigType
+} from '../BuildConfigType'
+import { ASCHAFFENBURG_ASSETS } from '../AssetsType'
+import mainImprint from './mainImprint'
+
+const APPLE_APP_ID = ''
+
+const commonAschaffenburgBuildConfig: CommonBuildConfigType = {
+  appName: 'Aschaffenburg App',
+  appIcon: 'app_icon_integreat',
+  lightTheme,
+  darkTheme,
+  assets: ASCHAFFENBURG_ASSETS,
+  cmsUrl: 'https://cms.integreat-app.de',
+  switchCmsUrl: 'https://cms-test.integreat-app.de',
+  shareBaseUrl: 'https://aschaffenburg.app',
+  allowedHostNames: ['cms.integreat-app.de', 'cms-test.integreat-app.de'],
+  internalLinksHijackPattern: 'https?:\\/\\/(cms(-test)?\\.integreat-app\\.de|web\\.integreat-app\\.de|integreat\\.app)(?!\\/[^/]*\\/(wp-content|wp-admin|wp-json)\\/.*).*',
+  featureFlags: {
+    pois: false,
+    newsStream: true,
+    pushNotifications: true,
+    introSlides: true,
+    sentry: true,
+    developerFriendly: false
+  },
+  aboutUrls: {
+    default: 'https://integreat-app.de/about/',
+    en: 'https://integreat-app.de/en/about/'
+  },
+  privacyUrls: {
+    default: 'https://integreat-app.de/datenschutz/',
+    en: 'https://integreat-app.de/en/privacy/'
+  }
+}
+
+export const androidAschaffenburgBuildConfig: AndroidBuildConfigType = {
+  ...commonAschaffenburgBuildConfig,
+  splashScreen: true,
```

Comment:
Similar to malte this should be false as we don't want to design a separate splash screen :D